design-documents/graph-ql/coverage/company-credit.md

```diff
@@ -0,0 +1,39 @@
+# Queries
+
+```graphql
+
+###### Begin: Extending existing types ######
```

Comment:
I'm sure there's a reason, but why aren't we using the `extend type` syntax to make extensions explicit?

Reply:
This syntax is not supported by webonyx.